docs/references/astro/auth-store.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: $authStore
+description: Clerk's $authStore nanostore is a convenient way to access the current auth state.
+---
+
+# `$authStore`
+
+The `$authStore` store is a convenient way to access the current auth state. This store provides the minimal information needed for data-loading and helper methods to manage the current active session.
```

Comment:
It's taken from the `useAuth` docs but I think it's because the original state contains the client, session, user and organization resources but extracted out minimal properties only.

Reply:
I hadn't caught that in the `useAuth()` docs before. I would like to clarify a little here where possible -- just will tighten up the docs that much more.